test/runtests.jl

```diff
@@ -3,19 +3,13 @@ using Test
 
 using TikzPictures
 tp = TikzPicture("\\draw (0,0) -- (10,10);\n\\draw (10,0) -- (0,10);\n\\node at (5,5) {tikz \$\\sqrt{\\pi}\$};", options="scale=0.25", preamble="")
-save(TEX("test"), tp)
+td = TikzDocument([tp],["hello"])
```

Comment:
You can get better coverage if you create an empty `TikzDocument` and then add to it using the `push!` command supported by `TikzDocument`.

Reply:
Will fix now.